deployment/flatpak/usage.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Flatpak Usage
+subsection: flatpak
+section: deployment
+---
+
+# {{page.title}}
+
+## Installing Flatpak Applications
+
+### Setting up the Flathub repository: 
```

Comment:
I'd say these are too many nestings for something as simple ... sooo, skip the `## Installing Flatpak Applications` ?